File: ts/easy/rangeSum.ts
export const rangeSum = (a: number, b: number): number => {
  let result = 0;
  const start = a > b ? b : a;
  const end = a < b ? b : a;

  for (let i = start; i <= end; i++) {
    result += i;
  }
  return result;
};
console.log(rangeSum(3, 5));
//	O(n)

File: python/easy/range_sum.py
def range_sum(a,b):
	result = 0
	start = a
	end = b

	if a > b:
		start = b
		end = a

	for num in range(start, end+1):
		result += num
	return result
print(range_sum(3,5))
#	O(n)

